src/porepy/applications/test_utils/models.py

```diff
@@ -118,6 +118,91 @@ class Model(geometry, model_class):
     return model
 
 
+class BoundaryConditionTypes:
```

Comment:
Please comment on whether this class has overlap with existing ones.

Reply:
Not that I could find, no.